src/main/java/uk/gov/hmcts/reform/managecase/service/NoticeOfChangeQuestions.java

```diff
@@ -0,0 +1,191 @@
+package uk.gov.hmcts.reform.managecase.service;
```

Comment:
In ACA-72 there is a `noc` subdirectory within `service` directory - makes sense for this to be moved there ?

Reply:
sure will move now